src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -40,9 +40,11 @@ buildtype=DEBUG
 code_coverage=OFF
 build_tests=OFF
 generator="Unix Makefiles"
+source_dir="$1"
 __UnprocessedCMakeArgs=""
 
 for i in "${@:5}"; do
+    echo "\$i: $i"
```

Comment:
I guess this is just a remainder of a debugging output, could you please remove it?